client/app/domains/domains.controller.js

```diff
@@ -176,11 +117,9 @@ angular.module('App').controller(
      * Toggle the clicked domain checkbox
      * @param {string} domain name
      */
-    toggleDomain(domain) {
-      this.$scope.selectedDomains = _.xor(this.$scope.selectedDomains, [
-        domain,
-      ]);
-      this.atLeastOneSelected = this.$scope.selectedDomains.length > 0;
+    toggleDomain(domain, selectedDomains) {
+      this.selectedDomains = selectedDomains.map(({ name }) => name);
```

Comment:
I don't mind but some will prefer
```suggestion
       this.selectedDomains = _.map(selectedDomains, 'name');
```